libpdf/core.py

```diff
@@ -106,20 +110,17 @@ def main(  # pylint: disable=too-many-arguments,too-many-locals  # no reasonable
         if output_path:
             LOG.info("Output path: %s", output_path)
         else:
-            LOG.info("Writing extracted data to stdout")
-        LOG.info(
-            "Page range: [%s]", "all" if not pages else ",".join(str(x) for x in pages)
-        )
-        LOG.info(
-            "Page crop: %s",
-            "not cropped" if not page_crop else " ".join(str(x) for x in page_crop),
-        )
-        LOG.info("Smart page crop: %s", "on" if smart_page_crop else "off")
-        LOG.info("Extract annotations: %s", "no" if no_annotations else "yes")
-        LOG.info("Extract chapters: %s", "no" if no_chapters else "yes")
-        LOG.info("Extract paragraphs: %s", "no" if no_paragraphs else "yes")
-        LOG.info("Extract tables: %s", "no" if no_tables else "yes")
-        LOG.info("Extract figures: %s", "no" if no_figures else "yes")
+            LOG.info('Writing extracted data to stdout')
+        LOG.info('Page range: [%s]', 'all' if not pages else ','.join(str(x) for x in pages))
+        LOG.info('Page crop: %s', 'not cropped' if not page_crop else ' '.join(str(x) for x in page_crop))
+        LOG.info('Smart page crop: %s', 'on' if smart_page_crop else 'off')
+        LOG.info('Extract annotations: %s', 'no' if no_annotations else 'yes')
+        LOG.info('Extract chapters: %s', 'no' if no_chapters else 'yes')
+        LOG.info('Extract paragraphs: %s', 'no' if no_paragraphs else 'yes')
+        LOG.info('Extract tables: %s', 'no' if no_tables else 'yes')
+        LOG.info('Extract figures: %s', 'no' if no_figures else 'yes')
+        LOG.info('Extract rects: %s', 'no' if no_rects else 'yes')
+        LOG.info('Text rects crop: %s', 'no' if crop_rects_text else 'no')
```

Comment:
as @kreuzberger already mentioned, is this a typo?
```suggestion
        LOG.info('Text rects crop: %s', 'yes' if crop_rects_text else 'no')
```